extensions/ql-vscode/src/variant-analysis/data-flow-paths-view.ts

```diff
@@ -0,0 +1,62 @@
+import { ExtensionContext, ViewColumn } from "vscode";
+import { AbstractWebview, WebviewPanelConfig } from "../abstract-webview";
+import { assertNever } from "../pure/helpers-pure";
+import { telemetryListener } from "../telemetry";
+import {
+  FromDataFlowPathsMessage,
+  ToDataFlowPathsMessage,
+} from "../pure/interface-types";
+import { DataFlowPaths } from "./shared/data-flow-paths";
+
+export class DataFlowPathsView extends AbstractWebview<
+  ToDataFlowPathsMessage,
+  FromDataFlowPathsMessage
+> {
+  public static readonly viewType = "codeQL.dataFlowPaths";
```

Comment:
Should opening this view also be added as an activation event?

It might not be necessary to add it since it can only be opened if another CodeQL view is already open, but perhaps it would be good for consistency?

Reply:
Activation events are also used for opening a non-persisted view, so it would also be an activation event if you opened the view from somewhere else. However, as I mentioned I don't believe it's necessary for this view to be an activation event because another view will always have to be loaded already before you can open this one. However, all of our current views are in the activation events, even those that currently also have that behaviour (like `onWebviewPanel:resultsView`), so I think it would be good to add it for consistency.

Reply:
Seems odd to just do it for consistency but that's perhaps a tech-debt issue. Updated.